css/_abstract.scss

```diff
@@ -7,4 +7,38 @@ $blk-col:#000000;
  display: flex;
  justify-content: center;
  align-items: center;
+}
+// MEDIA QUERY MANAGER
+/*
+phone 0 - 600px;
+tab-port < 900px;
+tab-land < 1200px;
+normal < 1800px;
+big-desk > 1800px
+//default 1rem = 16px; 
+inside media query rem and em are same, set to browser defaule
+but research suggests em is better than rem in media query;
+*/
+@mixin respond ($break-point){
+    @if $break-point == phone {
+        @media (max-width: 37.5em) {//600px/16 
+            @content
+        };
+    }
+    @if $break-point == tab-port {
+        @media (max-width: 56.25em) {//900px/16
+            @content
+        };
+    }
+    @if $break-point == tab-land {
+        @media (max-width: 75em) {//1200px/16
+            @content
+        };
+    }
+    @if $break-point == big-desk {
+        @media (min-width: 112.5em) {//1800px/16
+            @content
+        };
+    }
+
```

Comment:
This seems like overkill Raj, I would try something simpler. If you want to store your breakpoints inside variables, that's good practice. For example:
```
$max-width-mobile: 767px;

@media (max-width: $max-width-mobile) {
    margin-left: 0;
  }
``` 
this is more semantic than `@include respond(phone) {`
With `@include respond(phone) {` I do not know if we are talking about a size smaller than a phone or bigger than a phone. If you use the term `max-` or `min-` it's easier to understand what's going on 🙂

Reply:
I'm not sure the `@if` statements are necessary either. Keep it simple!

Reply:
For this simple case, it is not necessary.